File: eco-core/core-plugin/src/main/java/com/willfp/talismans/command/CommandTalismans.java
package com.willfp.talismans.command;

import com.willfp.eco.core.EcoPlugin;
import com.willfp.eco.core.command.CommandHandler;
import com.willfp.eco.core.command.impl.PluginCommand;
import org.jetbrains.annotations.NotNull;

public class CommandTalismans extends PluginCommand {
    /**
     * Instantiate a new command handler.
     *
     * @param plugin The plugin for the commands to listen for.
     */
    public CommandTalismans(@NotNull final EcoPlugin plugin) {
        super(plugin, "talismans", "talismans.command.talismans", false);

        this.addSubcommand(new CommandReload(plugin))
        .addSubcommand(new CommandGive(plugin));
    }

    @Override
    public CommandHandler getHandler() {
        return (sender, args) -> {

        };
    }
}

File: eco-core/core-plugin/src/main/kotlin/com/willfp/talismans/command/CommandTalismans.kt
package com.willfp.talismans.command

import com.willfp.eco.core.command.impl.PluginCommand
import com.willfp.talismans.plugin
import org.bukkit.command.CommandSender

object CommandTalismans : PluginCommand(
    plugin,
    "talismans",
    "talismans.command.talismans",
    false
) {
    init {
        addSubcommand(CommandReload)
            .addSubcommand(CommandGive)
            .addSubcommand(CommandBag)
    }

    override fun onExecute(sender: CommandSender, args: List<String>) {
        sender.sendMessage(
            plugin.langYml.getMessage("invalid-command")
        )
    }

    override fun getAliases(): List<String> {
        return listOf("talis", "tal", "talismen", "talisman")
    }
}
